src/tree/SubscriptionTreeItem.ts

```diff
@@ -163,8 +163,8 @@ export class SubscriptionTreeItem extends SubscriptionTreeItemBase {
     ): Promise<AzExtTreeItem> {
         const experience = tryGetExperience(databaseAccount);
         const id: string = nonNullProp(databaseAccount, 'id');
-        const name: string = nonNullProp(databaseAccount, 'name');
-        const documentEndpoint: string = nonNullProp(databaseAccount, 'documentEndpoint');
+        const name: string = nonNullProp(databaseAccount, 'name', `of the database account ${databaseAccount.id}`);
+        const documentEndpoint: string = nonNullProp(databaseAccount, 'documentEndpoint', `of the database account ${databaseAccount.id}`);
```

Comment:
I met the error in this place today. every time when I open DB it shows in the console and doesn't catch properly. Is it an critical issue? Should we fix the cause or not? I think it might be an issue for users since I see the VSCode notification that something went wrong internally

Reply:
I get this error for the two sevoku-test accounts which failed to provision and lack the endpoint. Without the endpoint we can not connect. I'm wondering which other accounts are affected.